service/controller/resource/monitoring/remotewriteagentconfigsecret/test/case-1-awsconfig.golden

```diff
@@ -0,0 +1,12 @@
+data:
+  values: LSB1cmw6IGh0dHBzOi8vaHR0cHM6Ly9wcm9tZXRoZXVzL2FsaWNlL2FwaS92MS93cml0ZQogIG5hbWU6ICIiCiAgc2VuZGV4ZW1wbGFyczogbnVsbAogIHJlbW90ZXRpbWVvdXQ6ICIiCiAgaGVhZGVyczoge30KICB3cml0ZXJlbGFiZWxjb25maWdzOiBbXQogIG9hdXRoMjogbnVsbAogIGJhc2ljYXV0aDoKICAgIHVzZXJuYW1lOgogICAgICBsb2NhbG9iamVjdHJlZmVyZW5jZToKICAgICAgICBuYW1lOiBhZ2VudC1yZW1vdGUtd3JpdGUKICAgICAga2V5OiB1c2VybmFtZQogICAgICBvcHRpb25hbDogbnVsbAogICAgcGFzc3dvcmQ6CiAgICAgIGxvY2Fsb2JqZWN0cmVmZXJlbmNlOgogICAgICAgIG5hbWU6IGFnZW50LXJlbW90ZS13cml0ZQogICAgICBrZXk6IHBhc3N3b3JkCiAgICAgIG9wdGlvbmFsOiBudWxsCiAgYmVhcmVydG9rZW46ICIiCiAgYmVhcmVydG9rZW5maWxlOiAiIgogIGF1dGhvcml6YXRpb246IG51bGwKICBzaWd2NDogbnVsbAogIHRsc2NvbmZpZzogbnVsbAogIHByb3h5dXJsOiAiIgogIHF1ZXVlY29uZmlnOiBudWxsCiAgbWV0YWRhdGFjb25maWc6IG51bGwK
```

Comment:
We do not really see anything here. This does not bring much value.

Reply:
What would you like to have here? No tests?

Reply:
Having the content of the values would be enough I guess.

Reply:
That's encoded in base 64 by the secret creation mechanism. The code will not allow us to generate non base64 secret just for the tests

Reply:
Maybe we can have a totally different test that just test the part that generate the values before the secret then.